CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Change Log
 
+## [0.17.0] 11 June 2024
+
+### Added
+- Added radar sensor simulation
+  - Added API call to processes point cloud to create radar-like output
+    - `rgl_node_points_radar_postprocess`
+  - Added API call to remove ground using RANSAC method to fit the plane model to the point cloud
+    - `rgl_node_points_remove_ground`
+  - Added API call to filter ground based on the incident angle of the ray hitting the mesh triangle
+    - `rgl_node_points_filter_ground`
+  - Added API call to compact point cloud by given field (RGL_FIELD_IS_HIT_I32 or RGL_FIELD_IS_GROUND_I32)
+    - `rgl_node_points_compact_by_field`
```

Comment:
This may be on the same level as "Added radar sensor simulation", instead of being one of features introduced in radar sensor simulation.